R/regression.R

```diff
@@ -218,6 +219,30 @@ Regression <- function(formula,
         stop("'weights' and 'data' are required to have the same number of observations. They do not.")
     if (!is.null(subset) & length(subset) > 1 & length(subset) != nrow(data))
         stop("'subset' and 'data' are required to have the same number of observations. They do not.")
+    missing.variables <- apply(data, 2, function(x) all(is.na(x)))
```

Comment:
I'm aware that this function is already quite messy but there should be comments here to indicate the purpose of the code you added or it should be split into a separate function to improve readability.

Reply:
Made suggested changes in https://github.com/Displayr/flipRegression/pull/5